src/pages/Possibility/Possibility.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+	<www-page>
+		<router-view />
+	</www-page>
+</template>
+
+<script>
+import WwwPage from '@/components/WwwFrame/WwwPage';
+
+export default {
+	components: {
+		WwwPage
+	},
+	beforeCreate() {
+		const offset = 8 * 60 * 60 * 1000; // PDT is 8 hours behind UTC
+		const now = Date.now() - offset;
+
+		const dec3 = 1575331200000;
+		const dec14 = 1576281600000;
+		const dec26 = 1577318400000;
+		const jan1 = 157783680000;
+
+		let url = '/';
+		if (now >= dec3 && now <= dec14) {
```

Comment:
I believe your conditionals are not correct here. dec14, should display 12 days of lending, not giving tuesday.

Reply:
You're right, thanks